woocommerce/woocommerce-ios
# Product Creation AI: UI for preview screen

Closes: #10750


## Description

This PR adds UI for the preview screen of the product creation AI flow and updates navigation to display the screen.

## Testing instructions

- Log in to a WPCom store or a self-hosted store with Jetpack AI installed.
- Switch to Products tab and select "+" to add a new product.
- Select add product with AI.
- Enter product name and features to get to the preview screen.
- Notice that the UI matches the design. The values will need to be updated in a subsequent PR.

## Screenshots

| \ | Top details | Bottom details | Loading |
| - | - | - | - |
| Light mode | <img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/8c7d1ba2-9ee3-4ba3-b063-afe356766df6" width=320 /> | <img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/021c35b5-4cd5-45e4-889a-8dfc47db8582" width=320 /> | <img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/8d837440-cdda-4cbd-8bcc-e67d40553a23" width=320 /> |
| Dark mode | <img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/22ee783b-1795-469e-9c84-305683637177" width=320 /> | <img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/441a219b-523e-48a3-8117-0a370c724896" width=320 /> | <img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/7854550a-a560-4863-9ada-d069d249650a" width=320 /> |



---
